src/components/Main.tsx

```diff
@@ -279,7 +279,14 @@ export const Main = () => {
       {experimentConfig == null ? (
         <LoadingSplashScreen />
       ) : (
-        <NavigationContainer theme={reactNavigationTheme}>
+        <NavigationContainer
+          // This lets us dive into nested navigators using just a route name.
+          // We need to port to using the new syntax,
+          // so do `navigate('stackName', { screen: 'nestedSceneName' }`
+          // instead of `navigate('nestedSceneName')`:
+          navigationInChildEnabled
```

Comment:
Is this necessary on all instances of `Stack.Navigator` or other navigator component instances perhaps?

Reply:
Probably?